gradle/dependencies.gradle

```diff
@@ -132,7 +145,7 @@ libs += [
   jacksonAnnotations: "com.fasterxml.jackson.core:jackson-annotations:$versions.jackson",
   jacksonDatabind: "com.fasterxml.jackson.core:jackson-databind:$versions.jackson",
   jacksonDataformatCsv: "com.fasterxml.jackson.dataformat:jackson-dataformat-csv:$versions.jackson",
-  jacksonModuleScala: "com.fasterxml.jackson.module:jackson-module-scala_$versions.baseScala:$versions.jackson",
+  jacksonModuleScala: "com.fasterxml.jackson.module:jackson-module-scala_$versions.baseScalaForSpecialLibs:$versions.jackson",
```

Comment:
jackson-module-scala 2.13.0-rc2 is available for scala 3 - generally, older versions of jackson-module-scala will not be able to deal with scala 3 classes - a number of changes have gone into jackson-module-scala 2.13 branch to support scala 3 generated classes correctly